organization/documents.md

```diff
@@ -11,6 +11,7 @@ a full list of all of the papers connected to the [Community White Paper]({{ sit
 {:.table .table-hover .table-condensed .table-striped}
 | Title           | Purpose     | Download    | Source Link |
 | ------------- | ------------- | ------------ | ----------- |
+| The Critical Importance of Software for HEP | Input to the European Particle Physics Strategy Update 2025 | <https://doi.org/10.5281/zenodo.15097159> | [LaTeX](https://github.com/HSF/EPPSU-2025-Paper) |
```

Comment:
I see that the previous document is from 2020. I wonder if we should not be more inclusive and have as well the documents prepared by any of the activity areas? They are HSF, after all.

We have since 2020 the report from the HSF IRIS-HEP second analysis ecosystems workshop, the analysis facilities white paper, the PyHEP.dev 2024 workshop report. I may be missing others.

WDYT?

Reply:
I think it's a great idea - we just didn't update this list as much as we should have.

Do you have the details? Do you just want to add a commit to this branch?

Reply:
OK, cool. I do have the details (since an author on several). I will commit later today ...

Reply:
Done now :).